docs/howto/system_testing.md

```diff
@@ -720,66 +798,11 @@ elastic-package test system -v --tear-down
-to be started and enrolled in a given test Agent Policy in Fleet specifically to run those system tests.
-Once those tests have finished, this Elastic Agent is going to be unenrolled and removed from
-Fleet (along with the Agent Policies) and a new Elastic Agent will be created for the next test configuration
-file.
-
-These Elastic Agents can be customized adding the required settings for the tests in the test configuration file.
-For example, the `oracle/memory` data stream's `test-memory-config.yml` is shown below:
-```yaml
-vars:
-  hosts:
-    - "oracle://sys:Oradoc_db1@elastic-package-service-oracle-1:1521/ORCLCDB.localdomain?sysdba=1"
-agent:
-  runtime: docker
-  provisioning_script:
-    language: "bash"
-    contents: |
-      apt-get update && apt-get -y install libaio1 wget unzip
-      mkdir -p /opt/oracle
-      cd /opt/oracle
-      wget https://download.oracle.com/otn_software/linux/instantclient/214000/instantclient-basic-linux.x64-21.4.0.0.0dbru.zip && unzip -o instantclient-basic-linux.x64-21.4.0.0.0dbru.zip
-      wget https://download.oracle.com/otn_software/linux/instantclient/217000/instantclient-sqlplus-linux.x64-21.7.0.0.0dbru.zip && unzip -o instantclient-sqlplus-linux.x64-21.7.0.0.0dbru.zip
-      echo /opt/oracle/instantclient_21_4 > /etc/ld.so.conf.d/oracle-instantclient.conf && ldconfig
-      cp /opt/oracle/instantclient_21_7/glogin.sql /opt/oracle/instantclient_21_7/libsqlplus.so /opt/oracle/instantclient_21_7/libsqlplusic.so /opt/oracle/instantclient_21_7/sqlplus /opt/oracle/instantclient_21_4/
-  pre_start_script:
-    language: "sh"
-    contents: |
-      export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-""}:/opt/oracle/instantclient_21_4"
-      export PATH="${PATH}:/opt/oracle/instantclient_21_7:/opt/oracle/instantclient_21_4"
-      cd /opt/oracle/instantclient_21_4
-```
-
-Considerations for this mode of running Elastic Agents:
-- As this is in technical preview, it requires to define `ELASTIC_PACKAGE_TEST_ENABLE_INDEPENDENT_AGENT=true` environment variable to enable it.
-  ```shell
-  elastic-package stack up -v -d
-  cd /path/package
-  ELASTIC_PACKAGE_TEST_ENABLE_INDEPENDENT_AGENT=true elastic-package test system -v
-  elastic-package stack down -v
-  ```
-- Those packages that define in their manifest that require root privileges (`agent.privileges.root: true`) run these Elastic Agents.
-- If a package defines the `agent` folder of the Agent deployer, it will continue using the Elastic Agent as described in [section](#agent-service-deployer).
-- If a package that defines the agent service deployer (`agent` folder) wants to migrate to this mode of running Elastic Agents, these would be the steps:
-    - Create a new `_dev/deploy/docker` adding the service container if needed.
-    - Define the settings required for your Elastic Agents in all the test configuration files.
-
 #### Running system tests in parallel (technical preview)
```

Comment:
Should we remove this technical preview too ?

Reply:
As it is running for a couple packages in the integrations repo, probably it could be kept like this (as technical preview).

Reply:
Leave it as tech preview for now.